### PR TITLE
Updates to tutorial on improving ASR for specific words

### PR DESCRIPTION
**Pull request by svenchilton**
Removal of references to TAO, minor copy edits. Any broken links are due to tutorials not having been renamed in the main branch or not yet existing in that branch.

### CODE REVIEW

File: asr-improve-recognition-for-specific-words.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ To improve the recognition of specific words, use the following customizations.
...
+| 1. Word boosting                        | Quick and easy  | Temporarily extend the vocabulary while increasing the chance of recognition for a provided list of words.              | When you know that certain words or phrases are important.                                 | [Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-wordboosting.ipynb) |
+| 2. Custom vocabulary                    | Easy            | Permanently extend the default vocabulary to cover novel words of interest.                                             | When the default model vocabulary does not sufficiently cover the domain of interest.      | [Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-customize-vocabulary-and-lexicon.ipynb) |
+| 3. Custom pronunciation (Lexicon mapping)                      | Easy            | Explicitly guide the decoder to map one or more pronunciations (sequences of tokens) to a specific word                                          | When you know a word can have one or several pronunciations.                            | [Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-customize-vocabulary-and-lexicon.ipynb) |
```
Comment (LynseyFabel) [inline asr-improve-recognition-for-specific-words.md:27]:
```suggestion
| 3. Custom pronunciation (Lexicon mapping)                      | Easy            | Explicitly guide the decoder to map one or more pronunciations (sequences of tokens) to a specific word.                                          | When you know a word can have one or more pronunciations.                            | [Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-customize-vocabulary-and-lexicon.ipynb) |
```

File: asr-improve-recognition-for-specific-words.md [hunk 2]
```diff
@@ -22,11 +22,11 @@ To improve the recognition of specific words, use the following customizations.
...
+| 4. Retrain language model               | Moderate        | Train a new language model for the application domain to improve the recognition of domain specific terms.     | When domain text data is available.                        | [Training Tutorial]() |
+| 5. Fine tune an existing acoustic model | Moderately hard | Fine tune an existing acoustic model using a small amount of domain data to better suit the domain. | When transcribed domain audio data is available (10h-100h), and other easier approaches fall short.   | [Citrinet - Training Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-finetuning-citrinet-nemo.ipynb), [Citrinet - Deployment Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-deployment-citrinet.ipynb),<br> [Conformer-CTC - Training Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-finetuning-conformer-ctc-nemo.ipynb), [Conformer-CTC - Deployment Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-deployment-conformer-ctc.ipynb) |
```
Comment (LynseyFabel) [inline asr-improve-recognition-for-specific-words.md:29]:
```suggestion
| 5. Fine-tune an existing acoustic model | Moderately hard | Fine-tune an existing acoustic model using a small amount of domain data to better suit the domain. | When transcribed domain audio data is available (10h-100h), and other easier approaches fall short.   | [Citrinet - Training Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-finetuning-citrinet-nemo.ipynb), [Citrinet - Deployment Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-deployment-citrinet.ipynb),<br> [Conformer-CTC - Training Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-finetuning-conformer-ctc-nemo.ipynb), [Conformer-CTC - Deployment Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-deployment-conformer-ctc.ipynb) |
```

File: asr-improve-recognition-for-specific-words.md [hunk 3]
```diff
@@ -84,16 +84,16 @@ See the [notebook on lexicon mapping](asr-python-advanced-customize-vocabulary-a
 
 
 ## 4. Retrain language model
-A language model (LM) estimates the likelihood of observing a text sequence in the text corpus it is trained on. Introducing a new language model to an existing ASR pipeline is another approach to improve accuracy for niche settings. Riva supports n-gram language models trained and exported from either NVIDIA TAO Toolkit or KenLM. See Riva [documentation](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/asr/asr-customizing.html#training-language-models) for details.
+A language model (LM) estimates the likelihood of observing a text sequence in the text corpus it is trained on. Introducing a new language model to an existing ASR pipeline is another approach to improve accuracy for niche settings. Riva supports n-gram language models trained and exported from either NVIDIA NeMo or KenLM. See Riva [documentation](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/asr/asr-customizing.html#training-language-models) for details.
```
Comment (LynseyFabel) [inline asr-improve-recognition-for-specific-words.md:87]:
```suggestion
A language model (LM) estimates the likelihood of observing a text sequence in the text corpus it is trained on. Introducing a new language model to an existing ASR pipeline is another approach to improve accuracy for niche settings. Riva supports n-gram language models trained and exported from either NVIDIA NeMo or KenLM. For more information, refer to the [Riva documentation](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/asr/asr-customizing.html#training-language-models).
```

File: asr-improve-recognition-for-specific-words.md (hunk 2 above)
Comment (LynseyFabel) [inline asr-improve-recognition-for-specific-words.md:29]: The above tutorial links result in a 404 error (except for "Citrinet - Deployment Tutorial" and "Conformer-CTC - Training Tutorial"). I'm assuming once we go live, these tutorials will also.

**Review by LynseyFabel — APPROVED** [review]
I left a few comments in the .md file. My review is complete. Thank you!

File: asr-improve-recognition-for-specific-words.md (hunk 2 above)
Comment (svenchilton) [reply asr-improve-recognition-for-specific-words.md]: Some of the NeMo-based tutorials still need to be created (I haven't seen PRs for them), but yes

**Pull request merged**